go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
 	gopkg.in/yaml.v2 v2.4.0
+	gotest.tools/gotestsum v1.6.4 // indirect
```

Comment:
we probably don't need this

Reply:
Reruning `go mod tidy` should remove it. It was added through `go get` in Makefile right?